db/migrate/20230802170650_add_company_to_user.rb

```diff
@@ -2,6 +2,7 @@ class AddCompanyToUser < ActiveRecord::Migration[7.0]
   def change
     add_reference :users, :company, foreign_key: true
 
+    execute "INSERT INTO companies (name, created_at, updated_at) VALUES ('Subvisual', NOW(), NOW())"
     execute "UPDATE users SET company_id = 1;"
```

Comment:
```suggestion
    execute "UPDATE users SET company_id = (select id from companies limit 1);"
```

Haven't tested this, but something like this might make sense

Reply:
Why this suggestion? Because the id could not start from 1 or if there are more companies, but id `1` would be there the same no? 🤔

Reply:
Not necessarily. You might have a DB in which number 1 was deleted and then this would fail.

Reply:
Ohh ok, yea didn't think about that!